jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -136,7 +136,7 @@ def test_continuum_surrounding_spectral_subset(specviz_helper, spectrum1d):
     plugin.width = 3
 
     # Values have not yet been validated
-    np.testing.assert_allclose(float(plugin.results[0]['result']), 350.89288537581467, atol=1e-5)
+    np.testing.assert_allclose(float(plugin.results[0]['result']), 2.153181e-13, atol=1e-15)
```

Comment:
Because we're doing line flux unit conversion to all data, not just those from `JWST`, it has impacted these tests here. Because these values are marked as "not validated" as per the comment, I just copied over the results from the test and replaced them here, since it is suggested to me that the values don't actually matter. But I don't know if there is a better approach to this. If I know someone who will call me out and tell me there's a better way, it's @pllim !

Reply:
This is where science validation becomes important. Have the POs also test this plugin with some non-JWST data and see if results are correct. 

On our side:

1. Does it make sense to convert unit for arbitrary data that we do not fully understand?
2. We need to add some meaningful tests that actually preserve the scientific integrity of this plugin. Like it or not, someone is going to try to copy paste the value onto their papers.

Reply:
Also, do you understand this change? What was the unit before, what is the unit now?

Reply:
This is definitely a PR that requires PO review, so I agree for the first, and last, point. Though I wonder whether a full scientific integrity test suite should be a blocker for this PR or not?

As for your other point, the main solace I can provide for the "arbitrary data" is that the conversion only happens when the units are specifically in the right bases. Could a user theoretically provide some other data that happens to have "the same unit" but mean something different? I don't know

Reply:
> I wonder whether a full scientific integrity test suite should be a blocker for this PR or not?

Well, if we want *full* QA, this is never going to go in. So I would say no, but we first need to understand why our existing test results have changed and is the change correct.

Reply:
@pllim  (1) Regarding arbitrary units, these shouldn't be touched by this PR.  Specutils should know how to handle the units commonly used by astronomers:  Jy, erg/s/cm^2/A, etc..., regardless of what instrument/telescope is used.
(2) The easiest meaningful test is a  Gaussian with flux = 1 W/m^2.  Not necessary to test with real data if this gives the right answer.

Reply:
Here are the only two cases for spectra with physical  units:
(1) energy/time/area/wavelength[/solid_angle]  vs. wavelength or frequency--- 
     integrate over wavelength, convert to W/m^2[/sr]
(2) energy/time/area/frequency[/solid_angle] vs. wavelength or frequency--- 
    integrate over frequency, convert to W/m^2[/sr]